artemis-log-annotation-processor/src/main/java/org/apache/activemq/artemis/logs/annotation/LogBundle.java

```diff
@@ -27,7 +27,13 @@
 
    String projectCode();
 
-   /** if set, every code generated must match this regular expression.
-    *  this is useful to validate ranges.*/
+   /* If set, every code generated must match this regular expression.
+    * This is useful to validate ranges.
+    */
```

Comment:
This seems like it should stay Javadoc? Similar for new one below.

Reply:
Fixed.